### ISSUE TITLE
Save failure on all pages

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#336]
I went to https://forums.zotero.org/discussion/91091/parent-item-cgt3szi7-cannot-be-a-child-item and tried to save to Zotero. It said:

```
Saving failed
You can still save this page as a webpage item.
```

It should save with the Embedded Metadata translator, and does for me on desktop.

**Comment by dstillman** [comment]
Or maybe it's just saying that everywhere? I get that on https://www.zotero.org too, and that doesn't even have a translator — that's just a webpage save.

**Comment by dstillman** [comment]
I'm getting this across the board. Not sure if this is a regression in the build or a result of fixing the repo updates and something going wrong when the app pulled all the new/updated translators.

**Comment by michalrentka** [comment]
This was fixed with #333
